.circleci/config.yml

```diff
@@ -38,8 +38,8 @@ jobs:
       - run: yarn lint
       - run: yarn format:all
       - run: yarn tsc
-      - run: yarn test:all --coverage --runInBand
-      - run: '[ $COVERALLS_REPO_TOKEN ] && yarn coveralls < utils/storybook/coverage/lcov.info || true'
+      - run: yarn test:ci
+      - run: '[ $COVERALLS_REPO_TOKEN ] && yarn coveralls < .cache/coverage/lcov.info || true'
```

Comment:
Kind of a bummer .. over time we've lost our ability to bring up the coverage report via the deployment URL.

Reply:
That gives me an idea - maybe we can. I'll followup with a future PR to bring this back.